src/bionics.cpp

```diff
@@ -1277,6 +1283,24 @@ bool player::install_bionics(const itype &type)
         popup("invalid / unknown bionic id %s", bioid.c_str());
         return false;
     }
+    if (bioid == "bio_reactor" || bioid == "bio_advreactor") {
+		if (has_bionic("bio_furnace") || has_bionic("bio_storage") || has_bionic("bio_reactor") || has_bionic("bio_advreactor")) {
+			popup(_("You lack the required space in your torso!"));
```

Comment:
Little more descriptive: your X is getting in the way.  We don't track bionic "slots" so folks will justifiably wonder why they lack the space.